python/aiffairness/aifserver/__main__.py

```diff
@@ -71,4 +71,4 @@
         unprivileged_groups=args.unprivileged_groups
     )
     model.load()
-    kserve.ModelServer().start([model], nest_asyncio=True)
```

Comment:
looks like `nest_asyncio` is still required for explainers as it is using the asyncio loop here
https://github.com/kserve/kserve/blob/master/python/aixexplainer/aixserver/model.py#L51